xblock/test/test_plugin.py

```diff
@@ -52,6 +56,29 @@ def boom(identifier, entry_points):
         XBlock.load_class("bad_block", select=boom)
 
 
+@XBlock.register_temp_plugin(OverriddenBlock, "overridden_block", group='xblock.v1.overrides')
+@XBlock.register_temp_plugin(UnambiguousBlock, "overridden_block")
+def test_plugin_override():
+    # Trying to load a block that is overridden returns the correct override
+    override = XBlock.load_class("overridden_block")
+    assert override is OverriddenBlock
+
+
+@XBlock.register_temp_plugin(AmbiguousBlock1, "overridden_block", group='xblock.v1.overrides')
+@XBlock.register_temp_plugin(AmbiguousBlock2, "overridden_block", group='xblock.v1.overrides')
+@XBlock.register_temp_plugin(OverriddenBlock, "overridden_block")
+def test_plugin_override_ambiguous():
+
+    # Trying to load a block that is overridden, but ambigous, errors.
+    expected_msg = (
+        "Ambiguous entry points for overridden_block: "
+        "xblock.test.test_plugin.AmbiguousBlock1, "
+        "xblock.test.test_plugin.AmbiguousBlock2"
+    )
+    with pytest.raises(AmbiguousPluginError, match=expected_msg):
+        XBlock.load_class("overridden_block")
+
+
```

Comment:
could you add one more test case to ensure that PluginMissing is raised when there is an xblock.v1.overrride but no xblock.v1 ?